### ISSUE TITLE
Not parsing emails 

### ISSUE THREAD

**Issue by Tengrom** [nheijmans/malzoo#11]
Code is not  parsing emails . Issue is because in emailworker.py in function "process" variable "Email" is a dict not str. When I fix it like below it started parsing : 

 ```
 if isinstance(Email['filename'], str):
                msg = email.message_from_string(Email['filename'])
            else:
                msg = Email['filename']

```

Also exception handlers need to improved . I spend few hours to find where is that issue .

**Comment by nheijmans** [comment]
Thank you for the reported bug! I'm going to check your findings and will come up with an update asap. Also put the exception handling improvement on the backlog

**Comment by nheijmans** [comment]
I applied your fix in 3c2dfdd

**Comment by runninbutt** [comment]
Bug fix is not sufficient. You need to modify line 45 as well, where the msg gets the parsed Email['filename']
